rocketpy/rocket/rocket.py

```diff
@@ -1124,6 +1159,145 @@ def add_parachute(
         self.parachutes.append(parachute)
         return self.parachutes[-1]
 
+    def add_air_brakes(
+        self,
+        drag_coefficient_curve,
+        controller_function,
+        sampling_rate,
+        clamp=True,
+        reference_area=None,
+        initial_observed_variables=None,
+        override_rocket_drag=False,
+        return_controller=False,
+        name="AirBrakes",
+        controller_name="AirBrakes Controller",
+    ):
+        """Creates a new air brakes, storing its parameters such as
```

Comment:
I believe this description should be updated to match the input names more closely, since they don't have a `trigger function`.

Reply:
Good catch!